keith-vscode/src/pasta/stpa-interaction.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ *
+ * Copyright 2023 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import * as vscode from 'vscode';
+import { handleWorkSpaceEdit } from '../util';
+
+/** Command identifiers that are provided by pasta. */
+const pastaCommands = {
+    getLTL: 'pasta.getLTLFormula',
+    // sendModelCheckerResult: 'pasta.sendModelCheckerResult'
```

Comment:
Is this old or do you still plan to do something with this?

Reply:
this will be used in the future for further functionalities. i will delete it for now